src/index.css

```diff
@@ -1,13 +1,195 @@
 :root {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans",
+    "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  --inactive-color: #eaeaea;
+  --button-color: #fcadad;
+  --hover-color: #fcadadc6;
+  --light-font: lightgray;
+  --box-color: #f2f0f0;
+  --box-shadow: 0.5rem 0.5rem 0px 0px black;
+  --max-width: 640px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  overflow-x: hidden;
+}
+
+#root,
+header {
+  width: 100%;
+}
+
+body,
+#root {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 16px;
+}
+
+header {
+  background-color: var(--hover-color);
+  padding: 2rem;
+  margin-bottom: 1rem;
+}
+
+main {
+  position: relative;
+  margin: 1rem;
+  padding: 0 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: var(--max-width);
+}
+
+h1,
+h2 {
+  font-weight: 300;
+  max-width: var(--max-width);
+  margin: 0 auto;
 }
 
-code {
+code,
+h3,
+textarea {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h3,
+label {
+  font-size: 1.2rem;
+  font-weight: 400;
+  color: black;
+  width: 100%;
+  padding-bottom: 1rem;
+}
+
+textarea {
+  height: 6rem;
+  width: 100%;
+  margin: 1rem 0;
+  padding: 1rem;
+}
+
+.thought-form,
+.thought {
+  position: relative;
+  border: 1px solid black;
+  box-shadow: var(--box-shadow);
+  padding: 1rem;
+  margin: 1rem 1rem 2rem 1rem;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: baseline;
+  font-size: 0.875rem;
+  overflow-wrap: anywhere;
+}
+
+.thought-form {
+  background-color: var(--box-color);
+  display: flex;
+  margin-bottom: 4rem;
+}
+
+.counter,
+.error-message {
+  margin-top: -1rem;
+}
+
+.counter {
+  width: 100%;
+  text-align: right;
+}
+
+.error {
+  color: red;
+  font-style: italic;
+}
+
+.submit-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: none;
+  border-radius: 1.5rem;
+  padding: 0.5rem;
+  margin: 0.5rem 0;
+  background-color: var(--button-color);
+  font-weight: 500;
+  color: black;
+}
+
+.thought-wrapper {
+  width: 100%;
+  max-width: 100svw;
+}
+
+.like-container {
+  display: flex;
+  align-items: center;
+  overflow-y: hidden;
+}
+
+.like-btn {
```

Comment:
With the animation you have on your like buttons it would really add to the user experience if you added a pointer cursor. To the submit button aswell. ✨

Maybe this was something you had already considered but chose not to add?